src/main/java/com/parasoft/demoapp/config/kafka/KafkaConfig.java

```diff
@@ -24,8 +24,8 @@
 public class KafkaConfig {
     public static final String DEFAULT_ORDER_SERVICE_REQUEST_TOPIC = "inventory.request";
     public static final String DEFAULT_ORDER_SERVICE_RESPONSE_TOPIC = "inventory.response";
-    @Getter @Setter private static String orderServiceRequestTopic = DEFAULT_ORDER_SERVICE_REQUEST_TOPIC;
-    @Getter @Setter private static String orderServiceResponseTopic = DEFAULT_ORDER_SERVICE_REQUEST_TOPIC;
+    @Getter @Setter private static String orderServiceSendToTopic = DEFAULT_ORDER_SERVICE_REQUEST_TOPIC;
```

Comment:
`orderServiceSendToTopic` is used for order service only. Inventory servie will use `DEFAULT_ORDER_SERVICE_REQUEST_TOPIC` and `DEFAULT_ORDER_SERVICE_RESPONSE_TOPIC ` since it is no  need to change the destinations of inventory service.